reqwest-tracing/src/reqwest_otel_span_builder.rs

```diff
@@ -112,6 +113,26 @@ fn get_header_value(key: &str, headers: &HeaderMap) -> String {
     format!("{:?}", headers.get(key).unwrap_or(header_default)).replace('"', "")
 }
 
+/// Similar to [`DefaultSpanBackend`] but also adds the `http.url` attribute to request spans.
+///
+/// [`TracingMiddleware`]: crate::middleware::TracingMiddleware
+pub struct SpanBackendWithUrl;
```

Comment:
Not too happy with the name

Reply:
I was thinking we could have `WithUrl<DefaultSpanBackend>` but the tracing macro is not composable at all :(

Reply:
😢